akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -970,6 +970,9 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
           else
             gossipTo(peer)
         case None ⇒ // nothing to see here
+          if (cluster.settings.Debug.VerboseGossipLogging)
+            log.debug("Cluster Node [{}] dc [{}] will not gossiping this round", selfAddress, cluster.settings.SelfDataCenter)
```

Comment:
"will not gossip" (you're the native speaker)?

Reply:
I'm from the north of england, doesn't count